specs/eventing/test-plan/broker-lifecycle-conformance.md

```diff
@@ -0,0 +1,187 @@
+# Broker Lifecycle 
+
+From: https://github.com/knative/specs/blob/main/specs/eventing/control-plane.md#broker-lifecycle
+
+
+From the Spec: 
+
+>> A Broker represents an Addressable endpoint (i.e. it MUST have a status.address.url field) which can receive, store, and forward events to multiple recipients based on a set of attribute filters (Triggers). 
+
+>> Triggers MUST be associated with a Broker based on the spec.broker field on the Trigger; it is expected that the controller for a Broker will also control the associated Triggers. 
+
+>> When the Broker's Ready condition is true, the Broker MUST provide a status.address.url which accepts all valid CloudEvents and MUST attempt to forward the received events for filtering to each associated Trigger whose Ready condition is true. As described in the Trigger Lifecycle section, a Broker MAY forward events to an associated Trigger destination which does not currently have a true Ready condition, including events received by the Broker before the Trigger was created.
+
+>> The annotation eventing.knative.dev/broker.class SHOULD be used to select a particular implementation of a Broker, if multiple implementations are available. It is RECOMMENDED to default the eventing.knative.dev/broker.class field on creation if it is unpopulated. Once created, the eventing.knative.dev/broker.class annotation and the spec.config field MUST be immutable; the Broker MUST be deleted and re-created to change the implementation class or spec.config. This pattern is chosen to make it clear that changing the implementation class or spec.config is not an atomic operation and that any implementation would be likely to result in event loss during the transition.
+
+
+
+# Testing Broker Lifecycle Conformance: 
+
+We are going to be testing the previous paragraphs coming from the Knative Eventing Spec. To do this we will be creating a broker checking its immutable properties, checking its Ready Status and then creating a Trigger that links to it by making a reference. We will also checking the Trigger Status, as it depends on the Broker to be ready to work correclty. We will be also checking that the broker is addresable by looking at the status conditions fields. Because this is a Control Plane test, we are not going to be sending Events to these components. 
```

Comment:
```suggestion
We are going to be testing the previous paragraphs coming from the Knative Eventing Spec. To do this we will be creating a broker, checking its immutable properties, checking its Ready status and then creating a Trigger that links to it by making a reference. We will also checking the Trigger Status, as it depends on the Broker to be ready to work correctly. We will be also checking that the broker is addressable by looking at the status conditions fields. Because this is a Control Plane test, we are not going to be sending Events to these components. 
```